lib/Finepack.coffee

```diff
@@ -45,10 +46,10 @@ module.exports = (data, opts = {}, cb) ->
   validation = report.validate input
   return report.requiredMessage(cb, input) if validation.required
 
-  # The default sort is alphabetically
-  # after that we move some esthetic fields positions.
-  input = sort input
+  # The default sort order is alphabetical.
+  input = sort input, opts.sortOptions
 
+  # After sorting, we move some fields in new positions for esthetic effect.
   for key in KEYWORDS.sort when input[key]?
```

Comment:
Do you mean refactoring the code that sorts by `KEYWORDS.sort` or just removing that comment?